c7n/offhours.py

```diff
@@ -277,10 +294,41 @@ def get_local_tz(self, parts):
             self.log.warning(
                 "filter:offhours unknown tz %s for %s" % (
                     tz_spec, parts))
-                    
+
             return None
         return tz
-    
+
+
+    def is_custom(self, parts):
+        if len(parts) == 3 and parts.get("on") and parts.get("off"):
```

Comment:
Looking back at this line I'm thinking this is better and cleaner for `is_custom`:
```
def is_custom(self, parts):
        return len(parts) == 3 and "on" in parts and "off" in parts
```